common/src/main/java/com/wynntils/mc/event/ServerResourcePackEvent.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.mc.event;
+
+import com.wynntils.core.events.EventThread;
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraftforge.eventbus.api.Event;
+
+public abstract class ServerResourcePackEvent extends Event {
+    /**
+     * Fired on receiving {@link net.minecraft.network.protocol.common.ClientboundResourcePackPacket}
+     */
+    @EventThread(EventThread.Type.ANY)
+    @Cancelable
+    public static class ServerResourcePackLoadEvent extends ServerResourcePackEvent {
+        private final String url;
+        private final String hash;
+        private final boolean required;
+
+        public ServerResourcePackLoadEvent(String url, String hash, boolean required) {
+            this.url = url;
+            this.hash = hash;
+            this.required = required;
+        }
+
+        public String getUrl() {
+            return url;
+        }
+
+        public String getHash() {
+            return hash;
+        }
+
+        public boolean isRequired() {
+            return required;
+        }
+    }
+
+    @EventThread(EventThread.Type.ANY)
+    public static class ServerResourcePackClearEvent extends ServerResourcePackEvent {}
```

Comment:
We used to have a convention where an inner class like this would be named only `Clear`, so the total name would be `ServerResourcePackEvent.Clear`. This convention is used in e.g. `RenderEvent`. 

Now it seems like most new classes has followed this new convention where the names just stack, and the full name becomes almost duplicated.

I'm not sure I like that. I kind of liked the old, shorter format as well, even if it meant that the actual event class name did not end in "Event". But I am open to other views as well. :-) At the very least, we should be consistent. 

Fixing this consistently is a separate issue, but I'll just want to note that what I envisioned when I suggested this was the shorter format. Not saying you should change anything, just trying to explain that I did not really think ` ServerResourcePackEvent.ServerResourcePackLoadEvent` was better than what was before. :-)

Reply:
Oh, I just forgot to rename these, don't worry about it 😅

Reply:
Ok, thanks, but my observation still stands -- we have a mix of both kinds of naming, and we should really do something about it...